docs/Streams.md

```diff
@@ -78,7 +78,7 @@ Typically, the buffer count is one, which means that most events will include a
 
 When the buffer count is 0, it signifies the reception of a QUIC frame with empty data, which also indicates the end of stream data.
 
-Currently, the maximum buffer count is 2 in the case of partial receive, where only a portion of the buffer data is consumed (as explained below). However, it is strongly advised not to assume in application code that the upper limit is always 2. This caution is important because future releases may incorporate multiple circular buffers to enhance performance, leading to potential changes in the buffer count limit.
```

Comment:
You shouldn't delete the "However, ...." section. It still applies. Also, we should state that it's 2, unless you enable multi-receive, and then it's 3.

Reply:
oh, here is where I added back by "add back buffer count caution" commit